gcc/rust/resolve/rust-ast-resolve-unused.h

```diff
@@ -43,14 +43,15 @@ class ScanUnused
 	}
       return true;
     });
+    return true;
   }
 
   static void Scan ()
   {
     auto resolver = Resolver::get ();
-    resolver->iterate_name_ribs ([&] (Rib *r) -> void { ScanRib (r); });
-    resolver->iterate_type_ribs ([&] (Rib *r) -> void { ScanRib (r); });
-    resolver->iterate_label_ribs ([&] (Rib *r) -> void { ScanRib (r); });
+    resolver->iterate_name_ribs ([&] (Rib *r) -> bool { return ScanRib (r); });
```

Comment:
Let's not change this as part of this PR. I think this scan unused needs some wor, in general, its causing dups for us at the moment.